docs/DeveloperGuide.md

```diff
@@ -847,40 +992,49 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+    1. Double-click the jar file Expected: Shows the GUI with a set of sample
```

Comment:
If im not wrong, in our v1.3, double-clicking the jar doesn't work properly!

Reply:
Seems to work with the current version!